dashboard/config/routes.rb

```diff
@@ -803,6 +803,7 @@
       get 'workshop_daily_survey/day/:day', to: 'workshop_daily_survey#new_daily_foorm'
       get 'workshop_pre_survey', to: 'workshop_daily_survey#new_pre_foorm'
       post 'workshop_survey/submit', to: 'workshop_daily_survey#submit_general'
+      get 'workshop_post_survey', to: 'workshop_daily_survey#new_post'
```

Comment:
Can you link the old pr where those two routes were consolidated?

Reply:
Oh yes sorry forgot to link that (it's in the PR description now): https://github.com/code-dot-org/code-dot-org/pull/59875/files#diff-3fffe83c5f87f14c4c59c00fc3556305f7d4a28ee7748b688700228b3c64e70d

Reply:
Thanks! So enrollment.rb is still set up to work in this old flow? Sorry just trying to make sure I understand the old pr considering it's kind of a partial revert/patch

Reply:
Yep they both just point to the same `workshop_daily_survey` method!